jxl/src/headers/extra_channels.rs

```diff
@@ -54,7 +54,7 @@ pub struct ExtraChannelInfo {
     // TODO(veluca93): if using Option<bool>, this is None when all_default.
     #[condition(ec_type == ExtraChannel::Alpha)]
     #[default(false)]
-    alpha_associated: bool,
+    pub alpha_associated: bool,
```

Comment:
same thing here on pub

Reply:
So what's the heuristic for knowing if it should be pub or not. Why is e.g. `dim_shift` pub?

Reply:
Done.

Reply:
I'd make dim_shift not pub either nowadays :D